apps/emqx/etc/emqx.conf

```diff
@@ -437,6 +440,47 @@ zones.default {
 
   }
 
+  authorization {
```

Comment:
why not merge up with authorization above?

Reply:
The authorization setting in zone can override the authorization at root level